pkg/bwagreement/bwagreement_test/common.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package testbwagreement
+package bwagreement_test
```

Comment:
don't rename this, `_test` usually means it is something that contains test not test utilities